src/Gui/CommandFeat.cpp

```diff
@@ -136,33 +136,47 @@ StdCmdSendToPythonConsole::StdCmdSendToPythonConsole()
 
 bool StdCmdSendToPythonConsole::isActive(void)
 {
-    return (Gui::Selection().size() == 1);
+    //active only if either 1 object is selected or multiple subobjects from the same object
+    int size = Gui::Selection().size();
+    int xsize = size > 0 ? Gui::Selection().getSelectionEx()[0].getSubNames().size() : 0;
+    return (size == 1 || (size >= 1 && size == xsize));

```

Comment:
Why so complicated? If you want to have a single object where an arbitrary number of sub-elements are selected then write:
`return Gui::Selection().getSelectionEx().size() == 1;`
And  your code is not even safe because Gui::Selection().size() is the number of selected objects of all documents while  Gui::Selection().getSelectionEx() returns the number of selected object of the active document.
So, a segfault can be easily achieved. Create a document, create an object and select it, create a new document => crash. This is because Gui::Selection().getSelectionEx() returns an empty vector.

Reply:
Yikes.  I'm glad you caught this now.